docs/documentation/configuration.html

```diff
@@ -0,0 +1,226 @@
+<head>
+    <title>EazyE2E Configuration</title>
+</head>
+<link rel="stylesheet" href="../main.css"/>
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" type="text/css">
+<ul class="topnav">
+    <li class="topnav"><a class="topnav" href="../about/about.html">About</a></li>
+    <li class="topnav"><a class="topnav" href="../contact/contact.html">Contact</a></li>
+    <li class="topnav topnav-active"><a class ="topnav" href="../documentation/documentation.html">Documentation</a></li>
+    <li class="topnav"><a class="topnav" href="../gettingStarted/gettingStarted.html">Getting Started</a></li>
+    <li class="topnav"><a class="topnav" href="../../index.html">Home</a></li>
+    <li><a class="home-text" href="../../index.html">EazyE2E</a></li>
+  </ul>
+<ul class="left-nav">
+    <li><a href="./EzElement.html">EzElement</a></li>
+    <li><a href="./EzGrid.html">EzGrid</a></li>
+    <li><a href="./EzGridItem.html">EzGridItem</a></li>
+    <li><a href="./EzList.html">EzList</a></li>
+    <li><a href="./EzListItem.html">EzListItem</a></li>
+    <li><a href="./EzRootElement.html">EzRootElement</a></li>
+    <li><a href="./EzText.html">EzText</a></li>
+    <li><a href="./EzKeyboardFunctions.html">EzKeyboardFunctions</a></li>
+    <li><a href="./EzMouseFunctions.html">EzMouseFunctions</a></li>
+    <li><a href="./EzLogMonitor.html">EzLogMonitor</a></li>
+    <li><a href="./EzLongSearch.html">EzLongSearch</a></li>
+    <li><a href="./EzPerformanceMonitor.html">EzPerformanceMonitor</a></li>
+    <li><a href="./EzProcess.html">EzProcess</a></li>
+    <li class="leftnav-active"><a href="./configuration.html">Configuration</a></li>
+</ul>
+
+<div class="main-documentation">
+    <h1>EazyE2E Configuration</h1>
+    <p>
+        EazyE2E provides the ability to configure various ways that the framework will function.  Any of the values 
+        below can be placed in the applications app.config file in the &lt;appSettings&gt; section.  Note that these 
```

Comment:
Revise "applications" to "application's"

Reply:
Changed text to "application's"